api/deploy/service.yaml

```diff
@@ -2,21 +2,21 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: api
+  name: api-service
   namespace: keptn
 spec:
   selector:
     matchLabels:
-      run: api
+      run: api-service
   replicas: 1
   template:
     metadata:
       labels:
-        run: api
+        run: api-service
     spec:
       containers:
-        - name: api
-          image: keptn/api:latest
+        - name: api-service
+          image: keptn/api-service:latest
```

Comment:
Not sure if the image change was on purpose, maybe we don't even need to do that? If we however do, we should also update the build process in .travis.yml

Reply:
As discussed, let's stay with the image name `keptn/api`.